garden_ai/pipelines.py

```diff
@@ -157,30 +158,24 @@ def __call__(
             endpoint = GardenConstants.DLHUB_ENDPOINT
 
         with globus_compute_sdk.Executor(endpoint_id=str(endpoint)) as gce:
-            # TODO: refactor below once the remote-calling interface is settled.
-            # console/spinner is good ux but shouldn't live this deep in the
-            # sdk.
-            with console.status(
-                f"[bold green] executing remotely on endpoint {endpoint}"
-            ):
-                future = gce.submit_to_registered_function(
-                    function_id=str(self.func_uuid), args=args, kwargs=kwargs
-                )
-                return future.result()
-
-    def _repr_html_(self) -> str:
-        style = "<style>th {text-align: left;}</style>"
-        title = f"<h2>{self.title}</h2>"
-        details = f"<p>Authors: {', '.join(self.authors)}<br>DOI: {self.doi}</p>"
-        optional = "<h3>Additional data</h3>" + tabulate(
-            [
-                (field, val)
-                for field, val in self.dict().items()
-                if field not in ("title", "authors", "doi", "steps") and val
-            ],
-            tablefmt="html",
-        )
-        return style + title + details + optional
+            future = gce.submit_to_registered_function(
+                function_id=str(self.func_uuid), args=args, kwargs=kwargs
+            )
+            return future.result()
+
+    # def _repr_html_(self) -> str:
+    #     style = "<style>th {text-align: left;}</style>"
+    #     title = f"<h2>{self.title}</h2>"
+    #     details = f"<p>Authors: {', '.join(self.authors)}<br>DOI: {self.doi}</p>"
+    #     optional = "<h3>Additional data</h3>" + tabulate(
+    #         [
+    #             (field, val)
+    #             for field, val in self.dict().items()
+    #             if field not in ("title", "authors", "doi", "steps") and val
+    #         ],
+    #         tablefmt="html",
+    #     )
+    #     return style + title + details + optional
```

Comment:
Should we just delete this and the imports that are unused now?

Reply:
Oh actually I left those commented out to remind myself to see if I could figure out how to keep it and keep dill happy once I had something working. 

That might as well be its own small issue, I'll open than and clean this up